deb/ubuntu-bionic/Dockerfile

```diff
@@ -1,6 +1,7 @@
 ARG GO_IMAGE
 ARG DISTRO=ubuntu
 ARG SUITE=bionic
+ARG VERSION_ID=18.04
```

Comment:
I wonder if it should be `1804` instead?

Reply:
Ah nevermind that's the right ID

Reply:
Oh! Missed your comment; it looks like `deb` / `dpkg` is happy with the current (CalVer) format;

```bash
dpkg --compare-versions "23.0.0~rc.1-0~ubuntu.22.10.0~kinetic" ">>" "23.0.0~rc.1-0~ubuntu.22.04.0~jammy" && echo "OK" || echo "KO"
OK

dpkg --compare-versions "22.10.0~kinetic" ">>" "22.04.0~jammy" && echo "OK" || echo "KO"
OK

dpkg --compare-versions "22.10" ">>" "22.04" && echo "OK" || echo "KO"
OK
```